### ISSUE TITLE
High level validation of JSON schema

### ISSUE THREAD

**Issue by leplatrem** [Kinto/kinto-admin#253]
It would be nice to have some sanity check for the JSON schema + ui:schema, for example the required fields consistency... Otherwise we have to go to the create record form to see the errors. 
Maybe a *preview panel* would help ?

**Comment by n1k0** [comment]
i'm planning to work on that after #371 lands. Basically, JSONEditor should perform JSON validation, to their consumers/wrappers.

**Comment by anselmos** [comment]
@leplatrem  - The #371 has been merged, is there any possibility for you to work on it? Or maybe there is anything that I could do to help with this ?

**Comment by leplatrem** [comment]
Unfortunately, I can't guarantee that this issue will be prioritized anytime soon on our side.

But if you're willing to work on this, it would be excellent! And I would be glad to guide you or review your contribution!

Thanks for your interest! Let us know :)
